sample-chat/src/main/res/values/strings.xml

```diff
@@ -40,6 +40,7 @@
     <string name="select_users_create_chat">Create new chat</string>
     <string name="select_users_edit_chat">Add Users to chat</string>
     <string name="select_users_get_users_error">Get users errors</string>
+    <string name="select_users_update_dialog_error">Update dialog error</string>
```

Comment:
Incorrect error message, we don't update dialog, we trying to get dialog from server